packages/tea-map-cache/src/tea.ts

```diff
@@ -37,7 +48,7 @@ async function getTeaBucketPath(params = {}) {
     },
     { retries, minTimeout: 1000, maxTimeout: 5000 }
   );
-}
+};
 
 module.exports = {
```

Comment:
This is no longer necessary

Reply:
I need to be more diligent about removing these.  Thanks.  761aac7db